DualityEditor/Utility/Extensions/ExtMethodsDualityFont.cs

```diff
@@ -36,47 +36,53 @@ public static class ExtMethodsDualityFont
 		/// </summary>
 		public static void RenderGlyphs(this DualityFont font, FontRenderGlyphCharSet extendedSet)
 		{
-			if (font.EmbeddedTrueTypeFont == null) throw new InvalidOperationException("Can't render glyphs of a Duality Font without embedded vector Font information.");
```

Comment:
Yes, this change was purely done to allow the property Kerning = true/false to recalculate. I can remove until we discuss kernings in more depth.